rplugin/python3/deoplete/__init__.py

```diff
@@ -63,7 +64,10 @@ def completion_begin(self, context):
         var_context['candidates'] = candidates
         self.vim.vars['deoplete#_context'] = var_context
 
+        # Set (and store) current &completeopt setting.  This cannot be done
+        # (currently) from the deoplete_start_complete mapping's function.
+        self.vim.command(
+            'call deoplete#mappings#_set_completeopt()')
```

Comment:
It must be called before `feedkeys()` instead of `complete()`?
I don't know the problem.  Please explain it.